.github/workflows/flowzone.yml

```diff
@@ -1351,10 +1351,7 @@ jobs:
       - custom_publish
       - custom_finalize
       - custom_clean
-    if: |
-      !failure() &&
-      needs.event_types.result == 'success' &&
-      needs.project_types.outputs.custom_always == 'true'
+    if: needs.project_types.outputs.custom_always == 'true'
```

Comment:
You still need the `always()` here if removing the `!failure()`, but then it should be good.